src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -191,19 +191,6 @@ public class AgentKnobs
             new EnvironmentKnobSource("AZP_USE_CREDSCAN_REGEXES"),
             new BuiltInDefaultKnobSource("false"));
 
-        // Task restrictions
-        public static readonly Knob TaskRestrictionsEnforcementMode = new Knob(
```

Comment:
I think we should be doing this, but it is worth calling out that this isn't without risk. It could break people who have flipped this knob to avoid the restrictions.

Reply:
Oh you're right, I forgot how we ordered that